### PR TITLE
feature(StockMovementToPatient)

### LINKED ISSUE #4309: Link the patient from the "Stock Movement Registry" if the entry is from a patient or the distribution is to a patient
It would be useful to link to a patient from the Stock Movement Registry if the flux is to a patient or from a patient. 

It would look like this:
image

### PR DESCRIPTION
**Pull request by lomamech**
- Add link from stock Movement to Patient

closes #4309

### CODE REVIEW

**Comment by lomamech** [comment]
g0A7K9vTc3

**Comment by lomamech** [comment]
@jniles all checks have passed

File: server/controllers/stock/core.js [hunk 1]
```diff
@@ -279,6 +279,7 @@ function getLotsMovements(depotUuid, params) {
     JOIN flux f ON f.id = m.flux_id
     LEFT JOIN document_map dm ON dm.uuid = m.document_uuid
     LEFT JOIN service AS serv ON serv.uuid = m.entity_uuid
+    LEFT JOIN patient AS pa ON pa.uuid = m.entity_uuid
```
Comment (jniles) [inline server/controllers/stock/core.js]: Can you instead use the `entity_uuid` to filter on patient?  That would save a join.  I'm pretty sure the `entity_uuid` here is actually the `patient.uuid`.

Comment (lomamech) [reply server/controllers/stock/core.js]: Okay

**Comment by lomamech** [comment]
@jniles it's done

**Review by jniles — APPROVED** [review]
Beautiful. 

bors r+

**Pull request merged**